apps/emqx/rebar.config

```diff
@@ -13,7 +13,7 @@
     , {jiffy, {git, "https://github.com/emqx/jiffy", {tag, "1.0.5"}}}
     , {cowboy, {git, "https://github.com/emqx/cowboy", {tag, "2.8.2"}}}
     , {esockd, {git, "https://github.com/emqx/esockd", {tag, "5.8.0"}}}
-    , {ekka, {git, "https://github.com/emqx/ekka", {tag, "0.10.0"}}}
+    , {ekka, {git, "https://github.com/emqx/ekka", {tag, "0.10.1"}}}
```

Comment:
Is it possible to delete app/emqx/rebar.config?

Reply:
I am not sure. It has the following comment:
```
%% This rebar.config is necessary because the app may be used as a
%% `git_subdir` dependency in other projects.
```
...So probably it's intentional.